### ISSUE TITLE
Add option for custom tileobject serialization

### ISSUE THREAD

**Issue by cosmiccoincidence** [RE-SS3D/SS3D#787]
## Summary

Basically title. This got left out of the tilemap rework but should be considered.

I *think* @henkhooft may already be on this task.

**Comment by Ryan089** [comment]
If BroodjeKipcorn is onto this already and this issue is just to track it, that's fine. If he's not, could you clarify *specifically* what you're after for this issue?

**Comment by cosmiccoincidence** [comment]
I agree it would be good to clarify it a bit more though I do not know the specifics. John and Broodje were talking about it when John reviewed the Tilemap PR recently.
